epix/clustering.py

```diff
@@ -249,7 +249,8 @@ def _cluster(x, y, z, ed, time, ci,
 classifier['parenttype'] = ['None', 'None', 'None', 'Kr83[9.405]','Kr83[41.557]', 'None', 'None']
 classifier['creaproc'] = ['None', 'None', 'None', 'None', 'None','None', 'None']
 classifier['edproc'] = ['ionIoni', 'hadElastic', 'None', 'None','None', 'None', 'None']
-classifier['A'] = [0, 0, 4, 0, 0, 0, 0]
+classifier['A'] = [np.iinfo(np.int16).max, np.iinfo(np.int16).max, 4, np.iinfo(np.int16).max,\
+    np.iinfo(np.int16).max, np.iinfo(np.int16).max, np.iinfo(np.int16).max]
```

Comment:
This makes it a bit simpler and more readable:

```suggestion
infinity = np.iinfo(np.int16).max
classifier['A'] = [infinity , infinity , 4, infinity , infinity , infinity , infinity ]
```

Reply:
Changed in: https://github.com/XENONnT/epix/pull/30/commits/eb7d44fd053a8331d5018eb070bf0a927de67988

Reply:
Just FYI, you can also just press commit suggestion.

Reply:
What? :-D Again learned something new about GitHub ;-)